### ISSUE TITLE
Can't access to che on openshift.io

### ISSUE THREAD

**Issue by cmoulliard** [openshiftio/openshift.io#536]
When I log using my username "cmoulliard@redhat.com" on openshift.io, I will be redirected to the openshift online server but I can't access Che Workspaces

![screenshot 2017-08-10 08 21 48](https://user-images.githubusercontent.com/463790/29157087-84b1485c-7da5-11e7-88a2-01fbd940518c.png)
![screenshot 2017-08-10 08 22 09](https://user-images.githubusercontent.com/463790/29157088-84b2b2fa-7da5-11e7-8bb0-f0880b3dddd7.png)
![screenshot 2017-08-10 08 22 24](https://user-images.githubusercontent.com/463790/29157089-84b661ca-7da5-11e7-8091-c66fa531854e.png)
![screenshot 2017-08-10 08 22 42](https://user-images.githubusercontent.com/463790/29157090-84b776c8-7da5-11e7-93fe-8f8afa87e186.png)
![screenshot 2017-08-10 08 23 01](https://user-images.githubusercontent.com/463790/29157091-84ca7354-7da5-11e7-842e-efb604a51e13.png)

**Comment by maxandersen** [comment]
@sbose78 any ideas why the login flow for @cmoulliard brings him to the openshift online starter console rather than openshift.io UI? 

And @cmoulliard not sure why you are concerned about Che workspaces via the runtime console - what you should expect to see is openshift.io main page with your spaces  defined (which would be empty as you haven't created any yet)

**Comment by maxandersen** [comment]
@cmoulliard I just noticed the spreadsheet with login info and I already put down there that your account is openshift online only, not openshift.io thus your login goes to openshift online not openshift.io.

We should see if we could handle that better, but until then, please create a separate account on openshift.io.

**Comment by joshuawilson** [comment]
@cmoulliard or you can ask @alexeykazakov to drop your OSO account and you can sign up again. It really depends if you want to keep your existing OSO account or not.

**Comment by alexeykazakov** [comment]
@maxandersen If unapproved user tries to login we currently redirect to the OSIO/OSO registration page. We set this redirect URL via an env var in fabric8-wit. And I agree It can be confusing. We should create a dedicated page in openshiftio app with some proper explanation what is wrong and what user can do about it and redirect to that page instead.
I created an issue in auth repo to track this work - https://github.com/fabric8-services/fabric8-auth/issues/53
